_maps/__MAP_DEFINES.dm

```diff
@@ -27,6 +27,8 @@
 	#define AI_OK "AI Allowed"
 	/// Ruins will spawn on this z-level
 	#define SPAWN_RUINS "Spawn Ruins"
+	/// A level that can be navigated to through space, but for real this time.
+	#define REACHABLE_SPACE_ONLY "Reachable Space Only"
```

Comment:
Do you think you could either rename this define or the existing `reachable` define to better clarify the distinction between the two?

Reply:
sure